src/foam/nanos/controller/ApplicationController.js

```diff
@@ -42,7 +42,6 @@ foam.CLASS({
     'foam.nanos.u2.navigation.FooterView',
     'foam.u2.stack.Stack',
     'foam.u2.stack.StackView',
-    'foam.u2.dialog.NotificationMessage',
```

Comment:
You might want to keep this. Otherwise we can't easily figure out that this class depends on 'foam.u2.dialog.NotificationMessage'